packages/arb-token-bridge-ui/src/util/EthDepositUtils.ts

```diff
@@ -59,15 +57,19 @@ export async function depositEthEstimateGas(
 
   // todo: update this when we support custom destination addresses for eth deposits
   const depositRequest = await ethBridger.getDepositRequest({
-    amount,
+    // amount is not needed because it won't revert if 0
+    // but some token contracts revert if 0 so we need amount for token gas estimation
+    amount: BigNumber.from(0),
     from: address
   })
 
-  const estimatedL1Gas = await l1Provider.estimateGas(depositRequest.txRequest)
+  const estimatedParentChainGas = await parentChainProvider.estimateGas(
+    depositRequest.txRequest
+  )
```

Comment:
no fallback for this - rpc should return gas estimate with 0 amount, the sender address and the deposit request data

will need a fallback value when we want to show gas estimate without wallet connection though